Lib/fontTools/merge.py

```diff
@@ -687,7 +756,10 @@ def merge(self, m, tables):
 	assert len(tables) == len(m.duplicateGlyphsPerFont)
 	for i,(table,dups) in enumerate(zip(tables, m.duplicateGlyphsPerFont)):
 		if not dups: continue
-		assert (table is not None and table is not NotImplemented), "Have duplicates to resolve for font %d but no GSUB: %s" % (i + 1, dups)
+		if table is None or table is NotImplemented:
+			log.warning("Have duplicates to resolve for font %d but no GSUB: %s" % (i + 1, dups))
```

Comment:
Can we offer any further advice? - esp something of the form "you should fix X or check on Y"

Reply:
I'll add some.